app/controllers/omniauth_controller.rb

```diff
@@ -70,6 +69,14 @@ def failure
 
 private
 
+  def multibyte_email_characters(email)
+    email.each_char.filter_map.with_index do |c, i|
+      next unless c.bytesize > 1
+
+      "Character #{i} = U+" + c.ord.to_s(16).upcase
```

Comment:
maybe worth a comment on how to convert from the outputted hex value in sentry, to the initial unicode string:
hex_value_string.to_i(16).chr(Encoding::UTF_8)